### ISSUE TITLE
Allow hot-loading?

### ISSUE THREAD

**Issue by DokiDerg** [kejonaMC/GeyserUpdater#100]
Could you add a option for if you have plugman where it will unload geyser, update, then load geyser again, this would be a great way to keep it updated without requiring a server restart very often

Example would be

``` java
// Load geyser plugin as a variable
Plugin plugin = PlugMan.getInstance().getPluginUtil().getPluginByName("Geyser-Spigot");
if (plugin == null) { return; } // Return if geyser is not found

// Unload Geyser
PlugMan.getInstance().getPluginUtil().unload(plugin)

// Do your update here

// Load Geyser
PlugMan.getInstance().getPluginUtil().unload(plugin)
```

**Comment by Konicai** [comment]
Geyser doesn't even support reloading, and reloading will likely be broken even further once Floodgate is merged into Geyser.

As much as I appreciate hotloading as a concept, Bukkit/Spigot/Paper have never even had a good reloading implementation, and I've never witnessed PlugMan behave in a way that suggests any improvements.

I'm sorry, but it's a hard no.
